### PR TITLE
Fix http events sink

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

File: app/src/main/java/io/apicurio/registry/events/http/HttpEventSink.java [hunk 1]
```diff
@@ -73,33 +69,35 @@ public void handle(Message<Buffer> message) {
...
+                    .header("ce-type", type)
+                    .header("content-type", MediaType.APPLICATION_JSON)
+                    .POST(HttpRequest.BodyPublishers.ofByteArray(data.getBytes()))
+                    .build();
+
+            final HttpResponse<String> eventResponse = getHttpClient()
+                    .send(eventRequest, HttpResponse.BodyHandlers.ofString());
+
+            if (eventResponse.statusCode() != 200) {
+                log.error("Error sending http event: {}", eventResponse.body());
```
Comment (paoloantinori) [inline app/src/main/java/io/apicurio/registry/events/http/HttpEventSink.java:88]: I don't have the whole context, so I might be saying the wrong thing. But I find surprising seeing a log call at `error` level without an explicit `fail()` or throw of exception.

Comment (carlesarnal) [reply app/src/main/java/io/apicurio/registry/events/http/HttpEventSink.java:88]: You're actually right, this should be a warning, not an error. Long day...

**Review by EricWittmann — APPROVED** [review]

**Comment by EricWittmann** [comment]
lgtm although I'm not 100% what the fix is... is it the use of `URI.create()` on line 74?

**Comment by carlesarnal** [comment]
> lgtm although I'm not 100% what the fix is... is it the use of `URI.create()` on line 74?

The URI.create and the fact that we're using now the java HTTP client instead of the vertx one.

**Pull request merged**